src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/ExpectedOutputs/JSON1001.SyntaxError.sarif

```diff
@@ -15,7 +15,7 @@
               },
               "messageStrings": {
                 "default": {
-                  "text": "at \"{0}\": JSON syntax error: {1}"
```

Comment:
>"text": "at \"{0}\": JSON syntax error: {1}" [](http://example.com/codeflow?start=18&length=44)

all the .Sarif changes are refresh baseline.
this one is not my change, 
I think it is just was not refreshed for a while.